tutorial/source/gmm.ipynb

```diff
@@ -116,21 +122,22 @@
      "output_type": "stream",
      "text": [
       "\n",
-      "...............................................................................\n",
-      "...............................................................................\n",
-      "......................................."
+      "...................................................................................................\n",
+      "...................................................................................................\n",
+      "...................................................................................................\n",
+      "...................................................................................................\n",
+      "..................................................................................................."
```

Comment:
?

Reply:
changed line length from 80 to 100; fixed typo; changed iters from 200 to 500.